src/main/java/com/google/api/codegen/transformer/ImportTypeTransformer.java

```diff
@@ -0,0 +1,32 @@
+/* Copyright 2016 Google Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.api.codegen.transformer;
+
+import com.google.api.codegen.viewmodel.ImportTypeView;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+public class ImportTypeTransformer {
+  public static List<ImportTypeView> generateImports(Map<String, String> imports) {
```

Comment:
The convention in the transformers is to have all the methods be instance methods, and have each transformer hold an instance of the transformers they use. Currently there is no important state or polymorphism used, but this pattern would make it easier to inject that in the future.